Source/santad/SNTPolicyProcessor.m

```diff
@@ -117,6 +121,14 @@ - (nonnull SNTCachedDecision *)decisionForFileInfo:(nonnull SNTFileInfo *)fileIn
     }
   }
 
+  if (csInfoError && csInfoError.code != errSecCSUnsigned &&
+      [[SNTConfigurator configurator] enableBadSignatureProtection]) {
```

Comment:
nit: Since the default is NO, this saves a few steps for most folks.
if ([[SNTConfigurator configurator] enableBadSignatureProtection] &&
     csInfoError && csInfoError.code != errSecCSUnsigned)